---
layout: page 
tags: [Jekyll, CAPRI, Docking, Scoring, Complexes, Assemblies, Protein, Structure]
modified: 2014-08-08T20:53:07.573882-04:00
comments: false
image:
  feature: pages/banner_resources.jpg
---

This page provide you links to various software tools, databases and web server which might be useful for CAPRI predictors.

#### Table of contents
{:.no_toc}
* table of contents
{:toc}

<br>
<HR>

### Contributing software and resources to this web site?

Do you want your software / database or web portal to be listed in the resources pages of our CAPRI web site?
One requirement is that it should be freely accessible to the community (at least for non-profit users).

Interested? Then please send the following information to capri.docking_[at]_gmail.com :

- Name of the software/resource
- Name of the group providing it (and web link to the group)
- Short test description of what it does / contains / offers
- web link to the software/resource
- Name and email of a contact person in case of problems with the resource (or alternatively web link of a support forum). 

Your request will be subjected to review by the CAPRI committee.

<br>
<HR>

### Software tools

  * [**FCC** - Fraction of common contact clustering](http://github.com/haddocking/fcc): Clustering of biomolecular complexes based on the fraction of common contacts. Contributed by the [BonvinLab](http://bonvinlab.org) (developed by João Rogrigues).

  * [**InterEvScore**](http://biodev.cea.fr/interevol/interevscore/): A scoring function for protein docking which combines a multi-body statistical potential with evolutionary information. This software was developed in the team of [Raphael Guerois](http://www.cea.fr/drf/ibitecs/english/Pages/units/sb2sm/lbsr/molecular-assemblies-genome-integrity.aspx).
 
  * [**KPAX**](http://kpax.loria.fr/): A protein structure aligner that calculates pair-wise, multiple, and flexible, structure alignments. Can also be used to search structural databases (e.g. SCOP, CATH, ECOD). Provides several output formats for visualisation in [Hex](http://hex.loria.fr) and [VMD](http://www.ks.uiuc.edu/Research/vmd/), and can be used to generate input scripts for homology modeling by [Modeller](https://salilab.org/modeller/). Kpax was developed by [Dave Ritchie](https://members.loria.fr/DRitchie/).

  * [**PDB-tools**](http://github.com/haddocking/pdb-tools):
  A collection of Python scripts for the manipulation (renumbering, changing chain and segIDs...) of PDB files. Contributed by the [BonvinLab](http://bonvinlab.org) (developed by João Rogrigues).


<br>
<HR>

### Servers

  * [**CCharPPI** : Scoring docking poses](https://life.bsc.es/pid/ccharppi/): Computational Characterisation of Protein-Protein Interactions. A compilation of around a hundred scoring functions for evaluating protein-protein complexes and models. Contributed by the [Fernandez-Recio lab](https://www.sbu.csic.es/research-groups/protein-interactions-and-docking/). [Contact](https://life.bsc.es/pid/ccharppi/info/contactus)

  * [**CPORT** : Prediction of protein-protein interface residues](http://milou.science.uu.nl/services/CPORT/). Contributed by the [BonvinLab](http://bonvinlab.org). For support, check the [HADDOCK BioExcel forum](http://ask.bioexcel.eu/c/haddock).

  * [**Map sequence conservation** onto a protein 3D structure](https://bmm.crick.ac.uk/~chalei01/mapconservation/). Contributed by the [laboratory of Paul Bates](https://www.crick.ac.uk/research/a-z-researchers/researchers-a-c/paul-bates/), Crick Institute, London UK. Contact: [raphael.chaleil@crick.ac.uk](mailto:raphael.chaleil@crick.ac.uk)

  * [**OPRA** : Predicting RNA binding sites in proteins](https://life.bsc.es/pid/opra). Contributed by the [Fernandez-Recio lab](https://www.sbu.csic.es/research-groups/protein-interactions-and-docking/). [Contact](https://life.bsc.es/pid/opra/info/contactus)

  * [**PRODIGY** : Prediction of the binding affinity of protein-protein complexes](http://milou.science.uu.nl/services/PRODIGY/). Contributed by the [BonvinLab](http://bonvinlab.org). For support, check the [HADDOCK BioExcel forum](http://ask.bioexcel.eu/c/haddock).

  * [**pyDockRescoring** : Scoring docking poses](https://life.bsc.es/pid/pydockrescoring/). Contributed by the [Fernandez-Recio lab](https://www.sbu.csic.es/research-groups/protein-interactions-and-docking/). [Contact](https://life.bsc.es/pid/pydockrescoring/info/contactus)

  * [**SpotON** : Determination of Hot-Spots at protein-protein interfaces](http://milou.science.uu.nl/services/SPOTON/). Contributed by the [BonvinLab](http://bonvinlab.org). For support, check the [HADDOCK BioExcel forum](http://ask.bioexcel.eu/c/haddock).

  * [**InterEvolAlign**](http://biodev.cea.fr/interevol/interevalign.aspx): An online tool to compute coupled multiple sequence alignments for a pair of proteins. This tool was developed in the team of [Raphael Guerois](http://www.cea.fr/drf/ibitecs/english/Pages/units/sb2sm/lbsr/molecular-assemblies-genome-integrity.aspx).

<br>
<HR>

### Performance of docking servers in CAPRI

|                     |      | | | Rank |
| SERVER              | URL  | 2017 | 2017 | 2016 | 2013 |
|                     |      | (a) | (b) | (c) | (d) |
|---------------------|---|:-:|:-:|:-:|:-:|
| CLUSPRO             | [<span style="color:#267CB9">https://cluspro.bu.edu</span>](https://cluspro.bu.edu) | 2 | 1/1  | 2 | 1 |
| DOCK/PIERR          | [<span style="color:#267CB9">http://clsb.ices.utexas.edu/web/dock.html</span>](http://clsb.ices.utexas.edu/web/dock.html) | NA  | 6 | 6 | 4# |
| GALAXYPPDOCK        | [<span style="color:#267CB9">http://galaxy.seoklab.org</span>](http://galaxy.seoklab.org) | NA | 5 | NA | NA |
| GRAMM-X             | [<span style="color:#267CB9">http://vakser.compbio.ku.edu/resources/gramm/grammx</span>](http://vakser.compbio.ku.edu/resources/gramm/grammx) | 9 | 6 | 4 | 9 |
| HADDOCK             | [<span style="color:#267CB9">https://haddock.science.uu.nl/services/HADDOCK2.2</span>](https://haddock.science.uu.nl/services/HADDOCK2.2) | 7 | 3/2 | 1 | 2 |
| HEXSEVER            | [<span style="color:#267CB9">http://hexserver.loria.fr</span>](http://hexserver.loria.fr) | NA | NA | NA | 5 |
| LZERD               | [<span style="color:#267CB9">http://kiharalab.org/proteindocking/pilzerd.php</span>](http://kiharalab.org/proteindocking/pilzerd.php) | 3 | 2/3 | 5 | 7 |
| MEGADOCK            | [<span style="color:#267CB9">http://www.bi.cs.titech.ac.jp/megadock</span>](http://www.bi.cs.titech.ac.jp/megadock) | NA | NA/3 | NA | NA |
| PATCHDOCK/FIBERDOCK | [<span style="color:#267CB9">http://bioinfo3d.cs.tau.ac.il/wk/index.php/Servers_%26_Software</span>](http://bioinfo3d.cs.tau.ac.il/wk/index.php/Servers_%26_Software) | NA | 6 | NA | 9/6** |
| PYDOCKWEB           | [<span style="color:#267CB9">https://life.bsc.es/servlet/pydock/home</span>](https://life.bsc.es/servlet/pydock/home) |   | 4/1 | NA | NA |
| SURFIT              | [<span style="color:#267CB9">https://sysimm.ifrec.osaka-u.ac.jp/docking/main</span>](https://sysimm.ifrec.osaka-u.ac.jp/docking/main) | NA | NA | NA | 8 |
| SWARMDOCK           | [<span style="color:#267CB9">https://bmm.crick.ac.uk/~svc-bmm-swarmdock</span>](https://bmm.crick.ac.uk/~svc-bmm-swarmdock) | 6 | 3/3 | 3 | 3 |
| TACOS               | [<span style="color:#267CB9">https://zhanglab.ccmb.med.umich.edu/TACOS</span>](https://zhanglab.ccmb.med.umich.edu/TACOS)  | NA | NA | NA | 9 |

  <i># Dock-PIE, an earlier version of Dock-PIERR</i>
  <br>
  <i>** The first number is the rank of PATCHDOCK, the second is the rank for FIBERDOCK</i>


The two numbers in Column 4, represent the rank for protein-protein docking predictions and for protein-peptide docking prediction respectively. 
NA, means that no docking predictions were evaluated for the corresponding server in the published assessment.

**References**

  (a) The challenge of modeling protein assemblies: The CASP12-CAPRI experiment. 
Lensink MF, Velankar S, Baek M, Heo L, Seok C, Wodak SJ. _Proteins._ Advanced Online Publication doi: 10.1002/prot.25419. (2017)

  (b) Modeling protein-protein and protein-peptide complexes: CAPRI 6th edition., Lensink MF, Velankar S, Wodak SJ. _Proteins._ 85:359-3778 (2017)

  (c) Prediction of homoprotein and heteroprotein complexes by proteins docking and template-based modeling: A Casp-CAPRI experiment.Lensink et al. _Proteins._ 84 Suppl 1:323-48 (2016)

  (d) Docking, scoring, and affinity prediction in CAPRI.
Lensink MF, Wodak SJ. _Proteins._ 81:2082-95 (2013)

<br>
<HR>

### Docking servers

  * [**HADDOCK**](http://milou.science.uu.nl/services/HADDOCK2.2): HADDOCK (High Ambiguity Driven protein-protein DOCKing) is an information-driven flexible docking approach for the modeling of biomolecular complexes. HADDOCK allows to include information from various sources, e.g. identified or predicted protein interfaces, as restraints to guide the docking process. HADDOCK can deal with a large class of modelling problems including protein-protein, protein-nucleic acids and protein-ligand complexes. A contribution from the [BonvinLab](http://bonvinlab.org). For support, check the [HADDOCK BioExcel forum](http://ask.bioexcel.eu/c/haddock).

  * [**InterEvDock**](http://bioserv.rpbs.univ-paris-diderot.fr/services/InterEvDock/): A docking server to predict the structure of protein-protein interactions using evolutionary information. This server was developed in the team of [Raphael Guerois](http://www.cea.fr/drf/ibitecs/english/Pages/units/sb2sm/lbsr/molecular-assemblies-genome-integrity.aspx).
 
  * [**pyDockWeb**](https://life.bsc.es/servlet/pydock/home): pyDockWEB is a web server for the structural prediction of protein-protein interactions. Given the 3D coordinates of two interacting proteins, pyDockWEB returns the best rigid-body docking orientations generated by FTDock and evaluated by pyDock scoring function, which includes electrostatics, desolvation energy and limited van der Waals contribution. Contributed by the [Fernandez-Recio lab](https://www.sbu.csic.es/research-groups/protein-interactions-and-docking/) Contact:[pydock@mmb.pcb.ub.es](mailto:pydock@mmb.pcb.ub.es)

<br>
<HR>

### Specialized docking servers

  * [**pyDockSAXS**](https://life.bsc.es/pid/pydocksaxs): Rigid-Body docking with experimental (SAXS) information. The server evaluates the capacity of each docking pose to describe the experimental curve of SAXS with the program CRYSOL. Contributed by the [Fernandez-Recio lab](https://www.sbu.csic.es/research-groups/protein-interactions-and-docking/). Contact info: [pydock@mmb.pcb.ub.es](mailto:pydock@mmb.pcb.ub.es)


### Conformational changes / sampling

  * [**ModFlex**](http://modflex.org/):
  A web resource developed by the Godzik's lab that provides information about structural changes undergone by the set of homologs in the PDB to a query sequence. The posted version is an _alpha_ version, which the Godzik group intend to develop further shortly. Any comments and suggestions for improvements will be most welcome and should be sent directly to Adam Godzik ([adam@sbpdiscovery.org](mailto:adam@sbpdiscovery.org)).

  * [**Contemplate**](http://bental.tau.ac.il/contemplate/):
  A web portal developed by the Nir BenTal's group that allows you to find alternate conformations by comparison with homologues in the PDB - can provide a list of template and/or modeled structures.

<br>
<HR>

### Databases and Benchmark datasets

  * [**CAPRI scoring decoys database**](http://cb.iri.univ-lille1.fr/Users/lensink/Score_set/): An expanded benchmark data set for testing scoring functions, comprising the ensemble of predicted complexes made available for scoring calculations in the CAPRI experiment. Contributed by the [Wodak](http://www.vib.be/en/research/scientists/Pages/Shoshana-Wodak-Lab.aspx) and [Lensink](http://ugsf-umr-glycobiologie.univ-lille1.fr/Presentation-430?lang=en) labs.

  * [**Dockground** benchmark sets](http://dockground.compbio.ku.edu): Benchmarks, decoys, models and other knowledge resources for docking. Contributed by the [Vakser](http://vakser.compbio.ku.edu/) lab.

  * [**InterEvol**](http://biodev.cea.fr/interevol/interevol.aspx): A database and web server designed for the evolutionary analysis of protein-protein interface structures. This database provides a list of structural homologs for each individual protein and a list of structural interologs (homologous complexes), as well as precomputed multiple sequence alignments for each interface. This database was developed in the team of [Raphael Guerois](http://www.cea.fr/drf/ibitecs/english/Pages/units/sb2sm/lbsr/molecular-assemblies-genome-integrity.aspx).

  * [**KBDOCK**](http://kbdock.loria.fr): A protein domain-domain interaction database for studying 3D domain-domain interactions and for proposing homology templates for protein docking. KBDOCK, was developed by [Anisah Ghoorah](https://sites.google.com/a/uom.ac.mu/anisah-ghoorah/), a former member of [Dave Ritchie's group](https://members.loria.fr/DRitchie/).

  * [**Protein-protein docking benchmark 5**](https://zlab.umassmed.edu/benchmark/). Contributed by the [Bates](https://www.crick.ac.uk/research/a-z-researchers/researchers-a-c/paul-bates/), [Bonvin](http://bonvinlab.org), Janin and [Zheng](https://www.umassmed.edu/zlab/) labs.

  * [**Protein-RNA docking benchmark v1.1**](https://life.bsc.es/pid/protein-rna-benchmark/): Composed of 106 protein-RNA systems, with 5 unbound-unbound, 4 unbound-pseudo-unbound, 62 unbound-bound, 5 unbound-model, 8 model-unbound, 19 model-bound cases and 3 model-model. Contributed by the [Fernandez-Recio lab](https://www.sbu.csic.es/research-groups/protein-interactions-and-docking/). [Contact](https://life.bsc.es/pid/pidweb/info/contact)

  * [**PPI4DOCK**](http://biodev.cea.fr/interevol/ppi4dock/): A large benchmark set for protein docking and scoring methods. It contains 1417 non-redundant docking targets between two subunits. Each subunit is a protein structural model obtained by homology modeling based on an unbound template. This database was developed in the team of [Raphael Guerois](http://www.cea.fr/drf/ibitecs/english/Pages/units/sb2sm/lbsr/molecular-assemblies-genome-integrity.aspx).

  * [**Protein-protein affinity database 2.0**](https://bmm.crick.ac.uk/~bmmadmin/Affinity/). Contributed by the [Bates](https://www.crick.ac.uk/research/a-z-researchers/researchers-a-c/paul-bates/), [Bonvin](http://bonvinlab.org), Janin and [Zheng](https://www.umassmed.edu/zlab/) labs.

  * [**SKEMPI**](https://life.bsc.es/pid/mutation_database/): Database of kinetics and energetics of mutant protein interactions.Contributed by the [Fernandez-Recio lab](https://www.sbu.csic.es/research-groups/protein-interactions-and-docking/).  [Contact](https://life.bsc.es/pid/mutation_database/contact.html)
  

